setup.py

```diff
@@ -53,10 +53,13 @@ def read(fname):
 
 install_requires = [
     'requests',
-    'boto3',
-    'google-cloud-storage',
 ]
 
+aws_deps = ['boto3']
+gcp_deps = ['google-cloud-storage']
+
+all_deps = install_requires + aws_deps + gcp_deps
```

Comment:
GCP is still young, we can afford to quietly remove it without annoying too many people.

```suggestion
all_deps = install_requires + aws_deps
```